index.html

```diff
@@ -0,0 +1,186 @@
+    <section class="work">
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio.png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Tonic</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(1).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Multi-Post Stories</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(2).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Tonic</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(3).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Multi-Post Stories</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+    </section>
+    
+    <!-- About section -->
+    <section class="about">
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/227613656-529df0be-c2d6-42dc-bb2d-688bc7bd1d5f.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/227613549-445b0393-971e-4312-9aab-e10cb5aa21c7.png)

Kindly add the missing border-radius so that our design matches the template in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Also kindly notice the difference in the background colors. `#ffffff` and `#e5e5e5`